client/Components/_shared/CohortSelect.js

```diff
@@ -37,14 +37,14 @@ CohortSelect.defaultProps = {
   handleChange: () => {}
 }
 
-const mapStateToProps = state => {
-  const { userDetails, cohorts, cohortUsers } = state
+const mapStateToProps = ({ cohorts }) => ({ cohorts })
```

Comment:
Hi, the intention for this select dropdown was to load just the cohorts associated to the logged in admin.

I think this would load all of the cohorts in our database.